app/models/membership.rb

```diff
@@ -1,6 +1,6 @@
 class Membership < ApplicationRecord
   belongs_to :supplier
-  belongs_to :user, optional: true
+  belongs_to :user
 
   validates :user_id, presence: true
```

Comment:
Since `:user` is now mandatory, can we also get rid of the `validates :user_id` line below?